query.go

```diff
@@ -70,7 +70,7 @@ func writeArgumentType(w io.Writer, t reflect.Type, value bool) {
 	default:
 		// Named type. E.g., "Int".
 		name := t.Name()
-		if name == "string" { // HACK: Workaround for https://github.com/shurcooL/githubv4/issues/12.
+		if name == "string" { // HACK: Workaround for https://github.com/shurcooL/githubv4/issues/12
```

Comment:
Could you @Laisky not include unnecessary changes like this?

Reply:
done